mods/ssb/moves.js

```diff
@@ -737,6 +737,7 @@ let BattleMovedex = {
 				carryOver.push({
 					hp: pokemon.hp / pokemon.maxhp,
 					status: pokemon.status,
+					statusTimer: pokemon.statusData.time,
```

Comment:
After doing some testing, it would be wiser to carry over all of `pokemon.statusData`. While sleep clause mod won't allow a player to put multiple foes asleep still, it will assume any sleeping pokemon was put to sleep by the other side, breaking things like sleep clause - rest (or nap time) interactions.

Reply:
that makes sense